vela/client.go

```diff
@@ -390,7 +390,7 @@ func (r *Response) populatePageValues() {
 // v is the HTTP response.
 //
 // For more information read https://github.com/google/go-github/issues/234
-func (c *Client) Call(method, u string, body interface{}, v interface{}) (*Response, error) {
+func (c *Client) Call(method, u string, body, v interface{}) (*Response, error) {
```

Comment:
Should we update the `u` variable in the function signature to be `url`?

```suggestion
func (c *Client) Call(method, url string, body, v interface{}) (*Response, error) {
```

I know this is outside the scope of your PR so I'm fine if we want to address it later 👍 

But it feels like a good opportunity to improve the function signature? 😃